README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is using meaningful tags to mark up your layout rather than just presentation. Like <section>,<p>,<footer>...that are indicated what they are used for.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+By using display:block, elements can take up the entire line by default, and you can set up height and width with them; but by using dispaly-inline, elements sit next to each other and they only take up the rooms that their contents needed and you can't set a height and width to them.
+
 3. What are the 4 areas of the box model?
 
+content area, padding area, border area and margin area.
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+cross axis
+
 5. Explain why git is valuable to a team of developers.
 
+Git is a version control system that used to track the changes you and your teammates made, making the collaboration easier.
+
```

Comment:
Very good answer! And you are very correct! Github is a way to make collaboration easier and more accessible!